schemas/account.js

```diff
@@ -10,6 +10,39 @@ const postAccounts = {
   }
 };
 
+const Pagination = {
+  title: 'bedrock-accounts-http pagination',
+  type: 'object',
+  required: ['email'],
+  additionalProperties: false,
+  properties: {
+    email: schemas.email(),
+    cursor: {
+      type: 'number',
+      minimum: 0
+    },
+    limit: {
+      type: 'number',
+      minimum: 10,
+      maximum: 1000
+    }
+  }
+};
+
+const Update = {
+  title: 'bedrock-accounts-http account update',
+  required: ['patch', 'sequence'],
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    patch: schemas.jsonPatch(),
+    sequence: {
+      type: 'number',
```

Comment:
there's actually an `integer` type that would be more appropriate here.  Use wherever is appropriate in schema updates here.

Reply:
cool

Reply:
This has not been addressed.

Reply:
somehow missed this will add it next go round right now working on the other 2 end points.